news/feature-3745.md

```diff
@@ -0,0 +1,17 @@
+`redis`: add batching support 
+
+Performance improvement is significant (at least +200%, but it peaked at almost +500%)
+
+I used the following configuration to enable batching:
+```
+destination d_redis {
+    redis(
+        host("localhost")
+        port(6379)
+        command("HINCRBY", "hosts", "$HOST", "1")
+        workers(8)
+        log-fifo-size(100000)
+        batch-lines(100)
```

Comment:
This also adds the `batch-timeout` option. Could you please also provide some information how you did that benchmark (configuration, setup).